src/factories/CommandFactory.ts

```diff
@@ -1,14 +1,17 @@
 import { commands_directory } from "../config.json";
 import Command from "../abstracts/Command";
 import DirectoryUtils from "../utils/DirectoryUtils";
+import { DEVELOPMENT_ENV } from "../config.json";
 
 class CommandFactory {
 	private commands: any = {};
 
 	async loadCommands(): Promise<void> {
+		const commandFileEnding = process.env.NODE_ENV === DEVELOPMENT_ENV ? "Command.ts" : "Command.js";
```

Comment:
Note that this is the same logic from `app.ts:24`. Why not extract this to an util? Something like...
```ts
class DirectoryUtils {
    appendFileExtension(fileName: string): string {
        const extension = process.env.NODE_ENV === DEVELOPMENT_ENV ? ".ts" : ".js";
        return fileName + extension;
    }
}